piton/tools/src/sims/manycore.config

```diff
@@ -25,7 +25,7 @@
     -model=manycore
     -toplevel=cmp_top
 #ifdef FLIST_BLACKPARROT
-    -flist=$DV_ROOT/design/chip/tile/blackparrot/bp_top/syn/flist.vcs
+    -flist=$DV_ROOT/design/chip/tile/blackparrot/bp_top/syn/piton_flist.vcs
```

Comment:
let's deal with the flist comment problem

Reply:
(rather than adding a new flist)

Reply:
The comment problem is dealt with already. We created a new flist for adding our mmio_cfg_loader. This usually sits with our tests flists.

Reply:
I'd rather not have an entire flist to maintain just for openpiton.  I would say add it to the normal top flist with a comment

// This is synthesizable for FPGAs, but should not be used for ASICs
// TODO: Replace with a minimal, FSM-based configuration engine

Reply:
You can also put multiple flists if you don't want to pollute yours upstream. It doesn't just have to be one on our end.